### PR TITLE
Feature/display memories

### LINKED ISSUE #5: Bedroom View
- A user should be able to click on the "Bedroom" from the birds eye view of the house

### LINKED ISSUE #6: Kitchen Component
As a user,
* I would like to select the "Kitchen" from the bird's-eye-view and select related memories.
    * Memories can be found by clicking on the stove

### LINKED ISSUE #9: Living Room
As a User,
* I would like to select the Living Room component from the bird's-eye-view
  * Once in the living room, I can click on an object (Yet to be determined), and view associated memories.
    * Ideally family photos

### LINKED ISSUE #11: Backyard view
- A user should be able to select the "Backyard" from the bird's-eye-view and select related memories.

### PR DESCRIPTION
**Pull request by Codo-Baggins**
## Description
- Added functionality to display multiple memories in a room
- Memories now are placed at specified coordinates predefined by their "x" and "y" properties
- Added functionality to open and close a Memory
- Added initial styling for the memory modal

## Motivation and Context
- We want this to be extremely accessible for our target audience

## How Has This Been Tested?
-  Everything thus far has been manual

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

## Additional Notes
- We need to go back and update the FE/BE contract: 
  -  1. Remove the `location` attribute on a memory
  -  2. Add the `x` and `y` attributes to a memory

### CODE REVIEW

**Review by Estaffieri — APPROVED** [review]
The X and Y coordinates is such a great idea and I believe solves quite a few problems. Thanks for updating that modal and refining the project.

File: src/Memory/Memory.js [hunk 1]
```diff
@@ -1,37 +1,87 @@
...
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  Modal,
+  Button,
+  TouchableOpacity,
+} from "react-native";
+// import Modal from "../UI/Modal";
```
Comment (brycemara) [inline src/Memory/Memory.js:12]: I think it was a good move using the built in Modal rather than this one.

File: src/Room/Room.js [hunk 2]
```diff
@@ -1,14 +1,58 @@
...
+            left: memory.x - 75,
+            width: 150,
+            height: 150,
+          };
+        }
+        return memoryCoordinates;
+      }, {});
+      return memoryCoordinates;
+    }
+  };
```
Comment (brycemara) [inline src/Room/Room.js:55]: Great stuff here with array prototypes!

**Review by brycemara — APPROVED** [review]
Good code. Big moves forward!!

File: src/Room/Room.js [hunk 3]
```diff
@@ -1,14 +1,58 @@
...
+              <Memory memory={memory} setSelectedMemory={setSelectedMemory} />
+            )}
+          </TouchableOpacity>
+        );
+      });
+      return memoryComponents;
+    }
+  };
+
+  const createMemoryStyles = () => {
```
Comment (cameronRomo) [inline src/Room/Room.js:33]: Very nice way of dynamically rendering the styles for each of the memories! Well done!

File: src/Room/Room.js [hunk 4]
```diff
@@ -1,14 +1,58 @@
...
+    } else {
+      let memoryCoordinates = memories.reduce((memoryCoordinates, memory) => {
+        if (!memoryCoordinates[memory.id]) {
+          memoryCoordinates[memory.id] = {
+            zIndex: 2,
+            position: "absolute",
+            borderRadius: 75,
+            borderWidth: 4,
+            borderColor: "red",
+            top: memory.y - 50,
```
Comment (cameronRomo) [inline src/Room/Room.js:45]: We should potentially check to see how this is rendered on a bigger/smaller tablet. Just curious if it still works.

**Review by cameronRomo — APPROVED** [review]
Very nice work with the dynamic nature of this PR.

**Pull request merged**